ezparking_client/src/components/header/index.js

```diff
@@ -2,13 +2,13 @@ import logo from '../../assert/logo.png';
 
 function PageHeader() {
     return ( 
-    
     <div className='inline-flex mx-auto my-3'>  
         <img className='h-10' src={logo} alt="Logo" />
```

Comment:
add a ```<a>``` tag to return home would be nice, be sure to clean the redirect state. Other than this enhancement, everything else looks great!